### PR TITLE
Feature/ignore unused

### PR DESCRIPTION
**Pull request by glvr182**
Cleaned up the repo a bit by deleting the dependencies since that is not really needed.
Also now the binary and log files will get ignored

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,19 @@
...
 TODO.md
+
+# Tests
 test/repos/repo
+
+# Binaries
+lazygit
+
+# Dependencies
+vendor/*
```
Comment (remyabel) [inline .gitignore]: I don't think it's a good idea to ignore the vendor directory, see [here](https://golang.github.io/dep/docs/FAQ.html#should-i-commit-my-vendor-directory). Updating dependencies should be a conscious decision.

Comment (glvr182) [reply .gitignore]: That's true, it's indeed a pro/con thingy. Let's see what the others think, idm either options.

Comment (jesseduffield) [reply .gitignore]: happy to merge the other changes to the .gitignore file though

Comment (glvr182) [reply .gitignore]: Good points, i'll take this PR down and create a new one without the vendor packages! @jesseduffield @remyabel

**Pull request closed without merging**